packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/DbSyncStakePoolProvider.ts

```diff
@@ -159,7 +159,7 @@ export class DbSyncStakePoolProvider extends DbSyncProvider implements StakePool
 
     // Get last epoch data
     const lastEpoch = await this.#builder.getLastEpochWithData();
-    const { poolOptimalCount, no: lastEpochNo } = lastEpoch;
+    const { optimalPoolCount, no: lastEpochNo } = lastEpoch;
```

Comment:
How did this not throw tsc errors?

Reply:
Thats our domain object, so it use to exists in our Epoch type, I just renamed it to make it consistent with the field in dbSync:

`optimal_pool_count	word64type	The optimal number of stake pools.`